src/test/java/com/coreos/jetcd/internal/impl/LoadBalancerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.coreos.jetcd.internal.impl;
+
+import com.coreos.jetcd.ClientBuilder;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.kv.PutResponse;
+import io.grpc.PickFirstBalancerFactory;
+import io.grpc.util.RoundRobinLoadBalancerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/**
+ * KV service test cases.
+ */
+public class LoadBalancerTest {
+
+  @Test
+  public void testPickFirstBalancerFactory() throws Exception {
+    KV client = ClientBuilder.newBuilder()
```

Comment:
I prefer to have following testing structures.
1. create client.
2. get *client.
3. do something.
4. close client.